src/controllers/trainee/routes.ts

```diff
@@ -1,7 +1,10 @@
 import { Router } from "express";
 import Controller from "./Controller";
+import { validateHandler } from "../../libs/routes";
+import validateConfig from "./validate";
+import { authMiddleWare } from "../../libs/routes";
 export const traineeRouter: Router = Router();
-traineeRouter.get("/", Controller.get);
-traineeRouter.post("/", Controller.post);
-traineeRouter.put("/", Controller.put);
-traineeRouter.delete("/", Controller.delete);
+traineeRouter.get("/", authMiddleWare('trainee1','read'), Controller.get);
```

Comment:
add the validateHandler middleware as well

Reply:
fixed it